packages/html-reporter/src/chip.tsx

```diff
@@ -19,7 +19,7 @@ import './chip.css';
 import './colors.css';
 import './common.css';
 import * as icons from './icons';
-import { clsx } from '@web/uiUtils';
+import { clsx } from '@playwright/web/src/uiUtils';
```

Comment:
In `playwright` we import stuff from `playwright-core` differently:
```
import { formatCallLog } from 'playwright-core/lib/utils';
```

Similarly, `@playwright/experimental-ct-core` imports from `@playwright/test` differently:
```
import { Watcher } from 'playwright/lib/fsWatcher';
```

Why is this one different?